### ISSUE TITLE
Loses trailing paren

### ISSUE THREAD

**Issue by avh4** [avh4/elm-format#19]
It looks like the outermost closing paren got moved to the first line in this example:

https://github.com/rtfeldman/dreamwriter/compare/0.16...elm-format#diff-cd8e6cfa12c24161b452f61a98434468L68

```
-            [		
-                g [		
-                    y <| toString yOffset		
-                ]		
-                {- SVG uses a painter algorithm, so we need axis at the end of		
-                   the list to keep bars from overlapping the axis, which gets		
-                   pretty ugly. -}		
-                (List.append		
-                    (List.map (\day -> bar day yOffset) days)		
-                    [axis])		
             ]
```

**Comment by avh4** [comment]
More minimal example:

```
ratio = graphHeight / (if range == 0 then 0.1 else toFloat range)
```

**Comment by avh4** [comment]
Fixed in 66074bf

**Comment by avh4** [comment]
Fixed for real in 0342d57.
